File: java/trip-service-kata/src/main/java/org/craftedsw/tripservicekata/exception/DependendClassCallDuringUnitTestException.java
package org.craftedsw.tripservicekata.exception;

public class DependendClassCallDuringUnitTestException extends RuntimeException {

	private static final long serialVersionUID = -4584041339906109902L;

	public DependendClassCallDuringUnitTestException() {
		super();
	}

	public DependendClassCallDuringUnitTestException(String message,
			Throwable cause) {
		super(message, cause);
	}

	public DependendClassCallDuringUnitTestException(String message) {
		super(message);
	}

	public DependendClassCallDuringUnitTestException(Throwable cause) {
		super(cause);
	}

	
}

File: c#_net5/TripServiceKata/TripServiceKata/Exception/DependendClassCallDuringUnitTestException.cs
using System;
using System.Runtime.Serialization;

namespace TripServiceKata.Exception
{
    [Serializable]
    public class DependendClassCallDuringUnitTestException : System.Exception
    {
        public DependendClassCallDuringUnitTestException() : base() { }

        public DependendClassCallDuringUnitTestException(string message, System.Exception innerException) : base(message, innerException) { }

        public DependendClassCallDuringUnitTestException(string message) : base(message) { }

        private DependendClassCallDuringUnitTestException(SerializationInfo info, StreamingContext context) : base(info, context) { }
    }
}
